BrainPortal/app/views/access_profiles/show.html.erb

```diff
@@ -87,7 +87,13 @@
   <% end %>
 
   <%= show_table(@access_profile, :edit_condition => check_role(:admin_user), :header => 'Users With This Profile') do |t| %>
-    <% user_names = (myusers.sort_by(&:login).map { |u| link_to_user_if_accessible(u) }.join(", ").html_safe.presence) || "(None)" %>
+
+    <% if !myusers.nil? && myusers.count > 0 %>
```

Comment:
`myusers` can never by nil, it will always be an array

why not use `myusers.present?` instead?